server/src/test/java/org/opensearch/index/shard/IndexShardTests.java

```diff
@@ -4901,10 +4906,44 @@ public void testRecordsForceMerges() throws IOException {
         closeShards(shard);
     }
 
-    private void populateSampleRemoteStoreStats(RemoteSegmentTransferTracker tracker) {
+    private void populateSampleRemoteSegmentStats(RemoteSegmentTransferTracker tracker) {
+        tracker.addUploadBytesStarted(10L);
```

Comment:
lets remove this duplicate line.

Reply:
Updated. Resolving this thread. Please reopen if needed.